src/Compilers/VisualBasic/Portable/Symbols/AssemblySymbol.vb

```diff
@@ -519,7 +519,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                         candidate = Nothing
                     End If
 
-                    If IsAcceptableMatchForGetTypeByNameAndArity(candidate) AndAlso Not candidate.IsHiddenByEmbeddedAttribute() AndAlso candidate <> result Then
+                    If IsAcceptableMatchForGetTypeByNameAndArity(candidate) AndAlso
+                        Not candidate.IsHiddenByVisualBasicEmbeddedAttribute() AndAlso
+                        Not candidate.IsHiddenByCodeAnalysisEmbeddedAttribute() AndAlso
```

Comment:
Yes. EmbeddedTypesInAnAssemblyAreNotExposedExternally

---
In reply to: [115869117](https://github.com/dotnet/roslyn/pull/18715#discussion_r115869117) [](ancestors = 115869117)